CMakeLists.txt

```diff
@@ -201,40 +204,23 @@ foreach(file ${SQL_FILES})
 	configure_file(${CMAKE_SOURCE_DIR}/migrations/cdserver/${file} ${PROJECT_BINARY_DIR}/migrations/cdserver/${file})
 endforeach()
 
+# Add system specfic includes for Apple, Windows and Other Unix OS' (including Linux)
+if (APPLE)
+	include_directories("/usr/local/include/")
+endif()
+
+# Load all of our third party directories
+add_subdirectory(thirdparty)
+
 # Create our list of include directories
 set(INCLUDED_DIRECTORIES
-	"dCommon"
-	"dCommon/dClient"
-	"dCommon/dEnums"
-
 	"dChatFilter"
 
-	"dGame"
-	"dGame/dBehaviors"
-	"dGame/dComponents"
-	"dGame/dGameMessages"
-	"dGame/dInventory"
-	"dGame/dMission"
-	"dGame/dEntity"
-	"dGame/dPropertyBehaviors"
-	"dGame/dPropertyBehaviors/ControlBehaviorMessages"
-	"dGame/dUtilities"
-
 	"dPhysics"
 
 	"dNavigation"
 	"dNavigation/dTerrain"
 
-	"dZoneManager"
-
-	"dDatabase"
-	"dDatabase/CDClientDatabase"
-	"dDatabase/CDClientDatabase/CDClientTables"
-	"dDatabase/GameDatabase"
-	"dDatabase/GameDatabase/ITables"
-	"dDatabase/GameDatabase/MySQL"
-	"dDatabase/GameDatabase/MySQL/Tables"
-
 	"dNet"
```

Comment:
Any particular reason we are keeping some of these include dirs here?

Reply:
Because any extra removals would require code changes or more work than necessary to avoid code changes.